client/gulpfile.js

```diff
@@ -28,6 +29,14 @@ const sources = {
     distPath: '../dist/assets/js/',
     all: 'js/**/*.js'
   },
+  jsHead: {
+    entry: './js/jsHead.js',
+    distPath: '../server/views/_partials/'
```

Comment:
Don't see why not, I'll move it.

Is it weird having nunjucks includes in the dist folder though? But I'm sticking in a TODO to revisit how we include inline js anyway.